README.md

```diff
@@ -1,24 +1,151 @@
-# README
+<div align="center">
+  <h1>Hello React Backend</h1>
+</div>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<a name="readme-top"></a>
 
-Things you may want to cover:
+<!-- TABLE OF CONTENTS -->
 
-* Ruby version
+# 📗 Table of Contents
 
-* System dependencies
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Hello React Frontend ](#-hello-react-frontend-)
+  - [🛠 Built With ](#-built-with-)
+    - [Key Features ](#key-features-)
+    - [Project BackEnd ](#project-backend-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-* Configuration
+<!-- PROJECT DESCRIPTION -->
 
-* Database creation
+# 📖 Hello React Frontend <a name="about-project"></a>
 
-* Database initialization
+> This is a single app with Rails as Backend
 
-* How to run the test suite
+## 🛠 Built With <a name="built-with"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+<h2> Tech Stack </h2>
+- Ruby on Rails
 
-* Deployment instructions
+<!-- Features -->
 
-* ...
+### Key Features <a name="key-features"></a>
+
+- Back end creating and api to store data and connect with the front end 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Project BackEnd <a name="frontend-repo"></a>
+
+- <a href="https://github.com/unorjikingsley/hello-react-front-end.git">Link to Back end repo</a>
```

Comment:
- [ ] As said the previous CR, please change this name to front-end repo as the link is not linking to the backend but to the frontend repo instead